spec/workers/ingest_w3c_log_worker_spec.rb

```diff
@@ -8,66 +8,50 @@
   end
 
   describe 'perform' do
-    before(:each) do
-      key = 'results.csv'
-      file = 'iis_w3c_example.log'
+    let(:bucket) {"bucket-name"}
+    let(:key) {"path/hits.log"}
+    let(:fixture) {"iis_w3c_example.log"}
+    let(:hash) {"abc123456def"}
```

Comment:
Can we maybe call this `etag` to be more consistent with how it's used?

Reply:
Oh I see why this is like this. Ignore this comment, I think `hash` is the right name.